.pre-commit-config.yaml

```diff
@@ -0,0 +1,16 @@
+exclude: ^((examples|other)\/.*\/.*\.(py|poscar|csv))
+repos:
+  - repo: https://github.com/mrakitin/exabyte-io-pre-commit-hooks
```

Comment:
Should be https://github.com/Exabyte-io/pre-commit-hooks, right? Based on https://github.com/Exabyte-io/pre-commit-hooks/pull/1/files

Reply:
Yes, eventually it should point to the https://github.com/Exabyte-io/pre-commit-hooks repo, preferably with a release as a `rev`.

Need to merge https://github.com/Exabyte-io/pre-commit-hooks/pull/2 to fully transition to that config. Right now it's still using my fork + commit hash.